cartography/intel/aws/ec2/internet_gateways.py

```diff
@@ -0,0 +1,83 @@
+import logging
+from typing import Dict
+from typing import List
+
+import boto3
+import neo4j
+
+from .util import get_botocore_config
+from cartography.util import aws_handle_regions
+from cartography.util import run_cleanup_job
+from cartography.util import timeit
+
+logger = logging.getLogger(__name__)
+
+
+@timeit
+@aws_handle_regions
+def get_internet_gateways(boto3_session: boto3.session.Session, region: str) -> List[Dict]:
+    client = boto3_session.client('ec2', region_name=region, config=get_botocore_config())
+    return client.describe_internet_gateways()['InternetGateways']
+
+
+@timeit
+def load_internet_gateways(
+    neo4j_session: neo4j.Session, data: List[Dict], region: str,
+    current_aws_account_id: str, update_tag: int,
+) -> None:
+    logger.debug("Loading %d Internet Gateways in %s.", len(data), region)
+    query = """
+            MERGE (ig:AWSInternetGateway{id: {InternetGatewayId}})
+            ON CREATE SET
```

Comment:
Is the `id` the same as the gateway's `arn`? If so, add `ON CREATE  SET ig.arn = {InternetGatewayId}`

Reply:
The ID is something like `igw-c0a643a9` and the Arn is in the format `arn:$partition:ec2:$region:$account:internet-gateway/$igw-id`. 

Do you want to build the Arn, or just leave it off if we're not receiving it verbatim from the API call?

Reply:
Hm.. I'm leaning toward building the ARN ourselves, but I don't have a super strong opinion - up to you!

Reply:
Actually I take that back - let's build the ARN so that we can load AWS tags for them basically effortlessly like this: https://github.com/lyft/cartography/blob/047a6bd1820e846e70bda4c6ddc18027da39e20a/cartography/intel/aws/resourcegroupstaggingapi.py#L49